### DISCUSSION TITLE
Doris 实际性能与官方文档性能测试不符，差距较大

### DISCUSSION THREAD

**Discussion by zhangtao106** [apache/doris#37259] [A - General / Q&A]
官方测试文档中60亿数据联表查询返回时间在150ms左右，我本地部署了三个BE，一个FE的集群，集群机器配置48C 256G，其中FE与一个BE在同一节点，两个表是unique 模型，A表6.5亿，有5列，B表31亿，有39列，执行SQL如下
select reg.province,reg.aaa,count(reg.id) as num
from (select province,aaa,id from td_sub_A where DATE_SUB(CURDATE(), INTERVAL 30 DAY) <= date(create_time))as reg 
left join td_sub_B_all as als
on als.province_name = reg.province and
    als.aaa = reg.aaa 
 group by reg.province,reg.aaa
 order by num desc
执行时间特别久，5000多秒没返回，超过时间限制后查询中断，执行过程中没发现异常日志，
官方测试数据
![Uploading image.png…]()
有没有大佬知道这种情况

**Comment by ixzc** [comment]
中文问题在ask.selectdb.com提个帖子

**Reply by zhangtao106** [reply]
好的
